### PR TITLE
Doc/update readme

### PR DESCRIPTION
**Pull request by mrrtree**
# Which issue does this PR close?

Closes #

# Rationale for this change
 
 

# What changes are included in this PR?

1. Add examples package, include create table, drop table, write and query
2. Update readme

# Are there any user-facing changes?

# How does this change test

### CODE REVIEW

File: utils/query.go [hunk 1]
```diff
@@ -11,7 +11,7 @@ import (
 
 func ParseQueryResponse(response *ceresdbproto.QueryResponse) ([]map[string]interface{}, error) {
 	if response.SchemaContent == "" {
-		return nil, errors.New("empty schema content")
```
Comment (jiacai2050) [inline utils/query.go:14]: Why this change?

Add comments when it will reach here

Comment (mrrtree) [reply utils/query.go:14]: done

File: utils/query.go [hunk 2]
```diff
@@ -10,6 +10,7 @@ import (
 )
 
 func ParseQueryResponse(response *ceresdbproto.QueryResponse) ([]map[string]interface{}, error) {
+	// schema is nil in sql CREATE Table or DROP Table
```
Comment (jiacai2050) [inline utils/query.go]: schema is nil when SQL is DDL(such as create/drop table)

Comment (mrrtree) [reply utils/query.go]: done

**Review by jiacai2050 — APPROVED** [review]
LGTM
